src/averageAges.js

```diff
@@ -15,11 +15,28 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => person.sex === 'm');
+
+  const menFromNeededCentury = men.filter(person =>
+    Math.ceil(person.died / 100) === century
+  );
+
+  let totalMenAge = 0;
+  let averageMenAge = 0;
```

Comment:
don't use `let` here, it's pretty much redundant if you'd use ternary operators

Reply:
so you can just reduce and calculate only on one needed array, without repeating yourself